packages/web-react/src/components/UNSTABLE_Slider/UNSTABLE_Slider.tsx

```diff
@@ -0,0 +1,84 @@
+import classNames from 'classnames';
+import React, { ForwardedRef, forwardRef } from 'react';
+import { useStyleProps } from '../../hooks';
+import { SpiritSliderProps } from '../../types';
+import { HelperText, ValidationText, useAriaIds } from '../Field';
+import { SLIDER_DEFAULT_PROPS } from './constants';
+import { useSliderStyleProps } from './useSliderStyleProps';
+
+const defaultProps = {
+  ...SLIDER_DEFAULT_PROPS,
+};
+
+/* We need an exception for components exported with forwardRef */
+/* eslint no-underscore-dangle: ['error', { allow: ['_UnstableSlider'] }] */
+const _UnstableSlider = (props: SpiritSliderProps, ref: ForwardedRef<HTMLInputElement>) => {
+  const propsWithDefaults = { ...defaultProps, ...props };
+  const {
+    'aria-describedby': ariaDescribedBy,
+    helperText,
+    id,
+    label,
+    max,
+    min,
+    step,
+    validationState,
+    validationText,
+    value,
+    ...restProps
+  } = propsWithDefaults;
+
+  const { classProps, props: modifiedProps } = useSliderStyleProps({ ...restProps, validationState });
+  const { styleProps, props: otherProps } = useStyleProps(modifiedProps);
+  const [ids, register] = useAriaIds(ariaDescribedBy);
+
+  const CSSVariable = '--slider-position';
+
+  const getSliderPosition = (num: number) => `${Math.round((100 * num) / 100)}%`;
+
+  const handleInput = (event: React.FormEvent<HTMLInputElement>) => {
+    const { target } = event as React.ChangeEvent<HTMLInputElement>;
+    const { value: inputValue } = target;
+
+    target.style.setProperty(CSSVariable, getSliderPosition(Number(inputValue)));
+  };
+
+  return (
+    <div {...styleProps} {...otherProps} className={classNames(classProps.root, styleProps.className)}>
+      <label htmlFor={id} className={classProps.label}>
+        {label}
+      </label>
+      <input
+        aria-describedby={ids.join(' ')}
+        className={classProps.input}
+        id={id}
+        onInput={handleInput}
+        ref={ref}
+        style={{ [CSSVariable]: `${getSliderPosition(value)}` } as React.CSSProperties}
+        type="range"
+        min={min}
+        max={max}
+        step={step}
+        value={value}
+      />
+      <HelperText
```

Comment:
Is the HelperText mandatory? The docs say the opposite.

Reply:
Helper text is not mandatory, the check for the prop is inside this component.